src/js/stores/MarathonStore.js

```diff
@@ -174,11 +177,15 @@ class MarathonStore extends GetSetBaseStore {
           this.emit(MARATHON_GROUP_CREATE_SUCCESS);
           break;
         case REQUEST_MARATHON_GROUP_DELETE_ERROR:
-          let groupErrorMessage = action.data;
-          if (!Object.keys(groupErrorMessage).length) {
-            groupErrorMessage = 'Error destroying group';
+          data = action.data;
+          if (!data || (Util.isObject(data) && !data.message)) {
```

Comment:
@wavesoft we should be using this https://github.com/dcos/dcos-ui/blob/master/src/js/events/MarathonActions.js#L64 which will extract the error from the XHR object.